portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormRules.path

```diff
@@ -112,11 +112,26 @@
 	<td></td>
 </tr>
 <!--RULE_BUILDER_AUTOFILL-->
+<tr>
+	<td>RULE_BUILDER_AUTOFILL_INPUT_FIELD</td>
```

Comment:
The xpath is same with `RULE_BUILDER_AUTOFILL_OUTPUT_FIELD`. I recommand to do some combile for those. And it would be great if we can define the locators with variable. That would be more readble and easy to maintain in the long run. We can have a call next week, I have some idea to do this.